leda/components/Rating/types.ts

```diff
@@ -27,10 +28,10 @@ export type SetCurrentSelected = React.Dispatch<React.SetStateAction<number>>;
 export type SetIsHovered = React.Dispatch<React.SetStateAction<boolean>>;
 
 export interface RatingProps {
-  /** Кастомизация иконки */
-  iconRender?: CustomRender<RatingProps, RatingState, IconProps>,
-  /** Выключенное состояние */
-  isReadOnly?: boolean,
+  /** Icon, default is Star */
+  icon?: IconTypes.Icons,
+  /** Don't click the rating */
+  isDisabled?: boolean,
   /** Количество звезд */
```

Comment:
Maybe translate to English as well?

Reply:
There is another task to translate all, so it will be done soon